src/main/java/scrolls/elder/logic/commands/PairCommand.java

```diff
@@ -118,6 +118,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         store.setPerson(personToPair1, newPerson1);
         store.setPerson(personToPair2, newPerson2);
+        model.commitDatastore();
         store.updateFilteredPersonList(Model.PREDICATE_SHOW_ALL);
```

Comment:
Would it be better to commit the Datastore after the personlist is reset? Then when someone undos they get the same view that they see when they previously ran the command.